docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../auth/scim/
+  - ../../enterprise/scim/
```

Comment:
Aliases are for redirects which I don't believe we need since this is a new page

https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases

```suggestion
```